purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesException.kt

```diff
@@ -9,5 +14,5 @@ open class PurchasesException(val error: PurchasesError) : Exception() {
         get() = error.underlyingErrorMessage
 
     override val message: String
-        get() = error.message
+        get() = overridenMessage ?: error.message
```

Comment:
By default we're always using the error code message. This will allow to overwrite that message in specific situations like the one in this PR.

Reply:
I think this makes sense!